jumpscale/packages/admin/frontend/data.js

```diff
@@ -156,6 +156,14 @@ const SOLUTIONS = {
     url: "/solutions/cryptpad_deploy",
     description: "CryptPad is the Zero Knowledge realtime collaborative editor."
   },
+  taiga: {
+    topic: "taiga_deploy",
+    type: "taiga",
+    name: "Taiga",
+    image: "./assets/taiga.png",
+    url: "/solutions/taiga_deploy",
+    description: "Build your all in one taiga solution"
```

Comment:
all what? :D

Reply:
all in one :D, means taiga integrated with threefold stuff

Reply:
ah ok. should be all-in-one?